### PR TITLE
Revert "Do not write log to job directory. (#841)"

### PR DESCRIPTION
**Pull request by Gerhut**
This reverts commit 4815f87d9d022292f1ad17d3a982ebb260f43435.

### CODE REVIEW

File: src/ClusterManager/job_manager.py [hunk 1]
```diff
@@ -333,12 +333,24 @@ def UpdateJobStatus(redis_conn,
...
+                                                 jobParams["dataPath"])
+    localJobPath = os.path.join(config["storage-mount-path"], jobPath)
+    logPath = os.path.join(localJobPath, "logs/joblog.txt")
+
+    if "userId" not in jobParams:
+        jobParams["userId"] = "0"
+
     if result == "Succeeded":
-        joblog_manager.extract_job_log(job["jobId"])
+        joblog_manager.extract_job_log(job["jobId"], logPath,
```
Comment (Anbang-Hu) [inline src/ClusterManager/job_manager.py]: This means writing to NFS upon success/failure even if elasticsearch is deployed. @xudifsd do you think it's ok?

Comment (xudifsd) [reply src/ClusterManager/job_manager.py]: This will be temporary, right?

Comment (Anbang-Hu) [reply src/ClusterManager/job_manager.py]: No, this will save to `/dlwsdata/work/<alias>/jobs/...../logs/joblog.txt`

Comment (xudifsd) [reply src/ClusterManager/job_manager.py]: I know, I meant this solution will be temporary, right? Then I'm fine with this.

**Pull request merged**